### ISSUE TITLE
Remove option to demote same power level

### ISSUE THREAD

**Issue by gnunicorn** [acterglobal/a3#1645]
Matrix doesn't allow us to demote someone's powerlevel is theirs is higher _or equal_ to ours. Thus instead of allowing that flow, as reported in https://github.com/acterglobal/a3/issues/1235#issuecomment-2063736934 and then giving the user an error message:
![IMG_0418](https://github.com/acterglobal/a3/assets/77483554/24077fee-6885-4b1f-b9b4-b4a6014dcdea) we should prevent this flow from starting and tell them why they can't do it.

**Comment by kumarpalsinh25** [comment]
@gnunicorn    
Is this issue is still persists? Can you check and confirm?

**Comment by gnunicorn** [comment]
no, this is still persistent. When the user has the ability to show power levels but not the powerlevel to change theirs, we still show the button and allow for that flow: https://github.com/acterglobal/a3/blob/ben-event-rsvp-notifications/app/lib/features/member/widgets/member_info_drawer.dart#L183-L193
